eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -69,6 +69,7 @@ jobs:
       - script: >
           dotnet test eng/service.proj
           --framework $(TestTargetFramework)
+          --filter "TestCategory!=Weekly&TestCategory!=Manually"
```

Comment:
For now can you limit this to only exclude "Manually" until we setup a daily and weekly split I'd like to avoid any confusion.

Reply:
Updated, thx